backend/config.loader.php

```diff
@@ -1,15 +1,12 @@
 <?php
 
-if (
-    !empty($_SERVER['REQUEST_URI']) &&
-    substr($_SERVER['REQUEST_URI'], 0, 8) == '/branch/'
-) {
-    // for Branches
-    require_once __DIR__.'/../../../backend/config.php';
-} else if ( is_readable(__DIR__.'/config.php') ) {
-    // for Configured MASTER
+if ( is_readable(__DIR__.'/config.php') ) {
+    // for configured local
     require_once __DIR__.'/config.php';
+} else if ( is_readable(__DIR__.'/../../_backend/config.php') ) {
```

Comment:
I feel like you're duplicating files too much. What would be different in the `_backend` folder than the `master/_backend` folder? Why not just have branches fall back to master?